__tests__/userService.test.js

```diff
@@ -10,19 +10,28 @@ jest.mock('../src/repository/userDAO');
 
 const mockDatabase = new Map();
 const mockUser1 = {
```

Comment:
I noticed that the mockUsers do not have a "class" "user" specified. It is not necessary, as the tests should function without the class specified. But, for consistency with our database, you could specify the class in each use.